x-pack/plugins/canvas/public/components/sidebar_header/custom_element_modal.tsx

```diff
@@ -0,0 +1,194 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+/* eslint-disable react/forbid-elements */
+import React, { PureComponent } from 'react';
+import { get } from 'lodash';
+import PropTypes from 'prop-types';
+import {
+  EuiModal,
+  EuiModalBody,
+  EuiModalFooter,
+  EuiOverlayMask,
+  EuiFormRow,
+  EuiFieldText,
+  EuiFlexGroup,
+  EuiFlexItem,
+  EuiButton,
+  EuiButtonEmpty,
+  EuiSpacer,
+  EuiTextArea,
+  EuiText,
+  EuiTitle,
+  EuiModalHeaderTitle,
+  EuiModalHeader,
+  EuiIcon,
+  // @ts-ignore EuiCard is an exported member of @elastic/eui
```

Comment:
nit: "is" or "is not"?

Reply:
I checked with the design team and it turns out `EuiCard` and `EuiFilePicker` in EUI have not been converted to TS yet which is causing the error. I updated the comment.